frontend/package.json

```diff
@@ -10,27 +10,19 @@
     "get-starknet": "^3.3.3",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-router-dom": "^6.26.2",
+    "react-router-dom": "^6.27.0",
     "react-scripts": "^5.0.1",
+    "react-toastify": "^10.0.6",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "jest",
+    "lint": "eslint .",
```

Comment:
also add separate command for prettier, so in the future both eslint and prettier can be run in the CI

Reply:
Prettier script was added. Please check it out. Thanks.